src/parser/sparqlParser/SparqlQleverVisitor.cpp

```diff
@@ -716,7 +717,7 @@ SparqlValues Visitor::visit(Parser::InlineDataFullContext* ctx) {
 }
 
 // ____________________________________________________________________________________
-vector<std::string> Visitor::visit(Parser::DataBlockSingleContext* ctx) {
+vector<TripleComponent> Visitor::visit(Parser::DataBlockSingleContext* ctx) {
   if (ctx->NIL())
     reportError(ctx,
```

Comment:
This can be `reportNotSupported(ctx, "Empty VALUES clauses are");`
(a little more concise, and leads to consistent error messages.